app/views/external_works/fetch.js.erb

```diff
@@ -2,14 +2,13 @@
   $j('#external_work_author').val("<%= @external_work.author %>").change();
   $j('#external_work_title').val("<%= @external_work.title %>").change();
   $j('#external_work_summary').val("<%= escape_javascript(@external_work.summary&.html_safe) %>").change();
-  $j('#external_work_summary').html("<%= escape_javascript(@external_work.summary&.html_safe) %>").change();
   $j('#fetched').val("<%= @external_work.id %>");
   $j('#external_work_rating_string').val("<%= @external_work.rating_string %>");
   var categories = <%= @external_work.category_strings.to_json.html_safe %>;
-  $j('input[name="external_work[category_strings][]"]').each(function() {
-    $j(this).prop("checked", categories.includes(this.value));
-  });
-  $j('#external_work_fandom_string').val("<%= @external_work.fandom_string %>");
-  $j('#external_work_relationship_string').val("<%= @external_work.relationship_string %>");
-  $j('#external_work_character_string').val("<%= @external_work.character_string %>");
+  $j.each(categories, function(i, category) {
+     $j("input[value='" + category + "']").prop('checked', true);
+   });
```

Comment:
I don't know if it matters, but if the user already has some categories checked, this won't uncheck them to match the categories of the retrieved external work.